solr/packaging/test/test_status.bats

```diff
@@ -33,9 +33,36 @@ teardown() {
   assert_output --partial "No Solr nodes are running."
   run solr start
   run solr status
-  assert_output --partial "Found 1 Solr nodes:"
+  assert_output --partial "running on port ${SOLR_PORT}"
   run solr stop
   run solr status
   assert_output --partial "No Solr nodes are running."
+}
+
+@test "status with --solr-url from user" {
+  solr start
+  run solr status --solr-url http://localhost:${SOLR_PORT}
+  assert_output --partial "\"solr_home\":"
+  solr stop
+}
+
+@test "status with --port from user" {
+  solr start
+  run solr status --port ${SOLR_PORT}
+  assert_output --partial "running on port ${SOLR_PORT}"
+  solr stop
+}
+
+@test "status with invalid --solr-url from user" {
+  solr start
+  run solr status --solr-url http://invalidhost:${SOLR_PORT}
+  assert_output --partial "Solr at http://invalidhost:${SOLR_PORT} not online"
+  solr stop
+}
 
+@test "status with --short format" {
```

Comment:
One question, and this isn't about this backport...   Is `--short` similar to `--quiet`(which we use in bin/solr start)?  Any other commands that *should* have a `--short` style flag?

Reply:
Well, `--short` is kind of feature creep here, my thought was to make it easier to run this cmd as part of scripts, where one URL per line can be parsed... But it could be removed too...

Reply:
Let's see how things evolve!  We can always change!